File: python/e00.branch.py
#!/usr/bin/python
# get input from user
response = raw_input("Would you like a toast? [Yes/No]: ")
# set response string using if/else construction
#   Test response to a string
if response == "Yes":
  response_string = "That's great"
else:
  response_string = "How about a muffin?"
# output the response string
print response_string

File: lessons/gen_scripts/ruby/scripts/e00.branch.rb
#!/usr/bin/env ruby
print "Would you like a toast? [Yes/No]: " # print prompt
$stdout.flush                              # flush unbuffered stream
response = gets.chomp                      # acquire string input (remove newline)

# branch on the string value using case/when
case response
when "Yes"
  response_str = "That's great!"
else
  response_str = "How about a muffin?"
end

puts response_str
